README.md

```diff
@@ -222,6 +222,27 @@ This invokes QEMU on the image in `builds/latest`.  It uses `-snapshot`,
 so any changes are thrown away after you exit qemu.  To exit, type
 `Ctrl-a x`.  For more options, type `Ctrl-a ?`.
 
+### Running with customizations
+
+Using `coreos-assembler`, it is possible to run a CoreOS style qcow2 image
+that was built elsewhere.  To do this, you need to use the `kola qemuexec`
+subcommand and point to the qcow2 image in the current directory.  (If you
+do not use the `--devshell` option, you will likely be unable to login to
+the image without providing an Ignition config.)
+
+```
+$ cosa kola qemuexec --devshell --qemu-image ./coreos-style.qcow2
+```
+
+Additionally, it is possible to pass in a custom Ignition config file when
+running an image.
+
+```
+$ cosa kola qemuexec --ignition ./ignition.json --ignition-direct --qemu-image ./coreos-style.qcow2
```

Comment:
```suggestion
$ kola qemuexec --ignition ./ignition.json --ignition-direct --qemu-image ./coreos-style.qcow2
```

Reply:
I don't really agree with this suggestion; all the other examples are using the `cosa` alias and I don't want to make an assumption that the user has the `kola` binary installed wherever they are.

Reply:
> and I don't want to make an assumption that the user has the kola binary installed wherever they are.

Right though about half of `cosa` would just fail if `kola` isn't there.

Reply:
> all the other examples are using the cosa alias and I don't want to make an assumption that the user has the kola binary installed wherever they are.

Ahh right, OK. I'll be honest I've probably used that alias once overall. :)

I personally think that for hacking on cosa, we should recommend the [pet container](https://github.com/coreos/coreos-assembler#installing-cosa-inside-an-existing-container) approach instead of using the alias. But anyway, point taken!